lib/media.js

```diff
@@ -71,8 +71,8 @@ function combineFileContents(fileArray, directory, filePrefix) {
     .then(function (fileContents) {
       const joined = fileContents.join('');
 
-      if (util.getByteLength(joined) > 50000) {
-        log('warn', 'Combined CSS contents are beyond the 50000 byte limit specified by AMPHTML.');
+      if (util.getByteLength(joined) > 75000) {
+        log('warn', 'Combined CSS contents are beyond the 75000 byte limit specified by AMPHTML.');
       }
```

Comment:
I wonder if you should put a second warning for articles > 70,000. So we can see how many articles are nearing the the limit too.